client/my-sites/site-settings/reading-rss-feed-settings/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import SettingsSectionHeader from 'calypso/my-sites/site-settings/settings-section-header';
+import { SyndicationFeedsSetting, SYNDICATION_FEEDS_OPTION } from './SyndicationFeedsSetting';
+
+type Fields = {
+	posts_per_rss?: number;
+};
+
+type RssFeedSettingsSectionProps = {
+	fields: Fields;
+	onChangeField: ( field: string ) => ( event: React.ChangeEvent< HTMLInputElement > ) => void;
+	handleSubmitForm: ( event: React.FormEvent< HTMLFormElement > ) => void;
+	disabled?: boolean;
+};
+
+export const RssFeedSettingsSection = ( {
+	fields,
+	onChangeField,
+	handleSubmitForm,
+	disabled,
+}: RssFeedSettingsSectionProps ) => {
+	const translate = useTranslate();
+	const { posts_per_rss } = fields;
+
+	return (
+		<>
+			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
+			<SettingsSectionHeader
+				title={ translate( 'RSS feed settings' ) }
+				showButton
+				onButtonClick={ handleSubmitForm }
+				disabled={ disabled }
+			/>
```

Comment:
What I think we can add here is the `isSaving` prop that will make sure the button label changes to "Saving..." when it is clicked.

This is a practice used with other existing settings sections.
```suggestion
			<SettingsSectionHeader
				title={ translate( 'RSS feed settings' ) }
				showButton
				onButtonClick={ handleSubmitForm }
				disabled={ disabled }
				isSaving={ isSavingSettings }
			/>
```

We could add similar prop to the `posts_per_page` setting introduced in the previous PR.

Reply:
Oh cool! I did not notice this one. Thanks :)